packages/commerce-sdk-react/src/hooks/useDNT.ts

```diff
@@ -24,11 +25,12 @@ interface useDntReturn {
 const useDNT = (): useDntReturn => {
     const auth = useAuthContext()
     const dntStatus = auth.getDnt()
+    const dntPreference = auth.getDntPreference()
     const updateDNT = async (preference: boolean | null) => {
         await auth.setDnt(preference)
     }
 
-    return {dntStatus, updateDNT}
+    return {dntStatus, dntPreference, updateDNT}
```

Comment:
Initially, I was thinking that customers could integrate DNT manually by using dntStatus and DEFAULT_DNT_STATE but it might be easier if dntPreference is directly given to them, which already takes DEFAULT_DNT_STATE/defaultDnt into account. Would changing this result in a major version update for the SDK?

Reply:
Yeah, I think this is ok change to make.